.env

```diff
@@ -0,0 +1,3 @@
+HOST=localhost
+PORT=8000
+DB_URL='mongodb+srv://bizimungu:0728038pp.PP@agakiriro.0xgn1.mongodb.net/blog?retryWrites=true'
```

Comment:
Your .env file should be ignore by git. i.e put it in .gitignore, otherwise everyone on the internet will be able see your secrets (passwords, ports, and keys) which is something that you don't wanna do.